### DISCUSSION TITLE
HSR could not initialize and always loop said start and 起動に失敗しました待機でもしますすべてのノードを停止します

### DISCUSSION THREAD

**Discussion by yani-rl-ai** [HumanSupportRobot/QandA#71] [Q&A] [answered]
my HSR could not wake up and always try to start and hear 起動に失敗しました待機でもしますすべてのノードを停止します
![Screenshot 2023-07-27 163831](https://github.com/HumanSupportRobot/QandA/assets/48663658/1168c1d6-dbc4-4dc9-acd7-c6ed2718e537)
![Screenshot 2023-07-27 163933](https://github.com/HumanSupportRobot/QandA/assets/48663658/a8a993b9-731c-4f7c-90fd-95d32b1e765a)

we tried to check the connection and do 
hsrb_mode 
rqt
plugins
we choose robot diagnostic and the result such as above figure. 
we have tried clean install and tried 
exxx_hash /dev/ttyCTI2 11
But there is no response.

we tried to diagnostic from http://hsrb.local/admin/ but the respond was very slow like not responding

please kindly help me

**Comment by yani-rl-ai** [comment]
Dear admin, please kindly help me

**Comment by hsr-admin** [comment, marked as answer]
HSR is waiting for communication with the 11 axes to be established, and at the timing when the stop button is released, communication is established and the ros node is activated.
I remember that this phenomenon may occur if communication with some amplifier is not established during the process.
Please check which axis of the amplifier is not communicating.
For example, check the power-on counts of the amplifiers, and if there is an axis that cannot be acquired, there is most likely a problem there.

The following is an example of command execution.
Execute the following command before releasing the stop button.

ROS_KEY_VALUES is the axis number and power ON count.

> journalctl -f -t rosdiag -o verbose "ROS_NAME=hsrb_robot_service: present_log_power_cycle"
Wed 2023-08-02 09:29:12.043075 JST [s=78141ad3993a406fbbbeeb70aa1cac0b;i=833b;b=6d14a9dd793443fe9f52b19665ed44d2;m=1741fe999;t=601e5c0ae30c0;x=ec96a7dfbc9d2933]
    _SYSTEMD_SLICE=system.slice
    _BOOT_ID=6d14a9dd793443fe9f52b19665ed44d2
    _MACHINE_ID=659a4c6e553b494f9510dcc33a791159
    _HOSTNAME=hsrb
    PRIORITY=7
    _TRANSPORT=journal
    _CAP_EFFECTIVE=0
    _SYSTEMD_CGROUP=/system.slice/containerd.service
    _SYSTEMD_UNIT=containerd.service
    _SYSTEMD_INVOCATION_ID=e56c4688c1e047d792b7ff4b9e74e84f
    _UID=500
    _GID=500
    _SELINUX_CONTEXT=docker-default (enforce)
    SYSLOG_IDENTIFIER=rosdiag
    _COMM=diagnostics
    _EXE=/opt/ros/noetic/lib/tmc_journald/diagnostics
    _CMDLINE=/opt/ros/noetic/lib/tmc_journald/diagnostics
    _PID=18396
    MESSAGE=hsrb_robot_service: present_log_power_cycle: exxx_read present_log_power_cycle succeeded
    ROS_NAME=hsrb_robot_service: present_log_power_cycle
    ROS_HARDWARE_ID=motor present_log_power_cycle
    ROS_TIME_STAMPS=1690936151372907161
    ROS_KEY_VALUES=11   4003
                   12   4003
                   13   4003
                   21   3984
                   22   4002
                   23   4001
                   24   3997
                   25   3997
                   31   4003
                   32   4003
                   41   4007
    _SOURCE_REALTIME_TIMESTAMP=1690936152043075

**Reply by yani-rl-ai** [reply]
okay thank you very much for your answer, i will try soon and let you know soon

**Comment by yani-rl-ai** [comment]
> 
i found this following result, please kindly check, the motor number 31 is UNKNOWN : 
-- Logs begin at Thu 2023-07-27 01:35:09 JST. --
Fri 2023-07-28 01:49:55.437367 JST [s=834792cb7b9a47528e9f1fea693fdc95;i=11671;b=2b42fb535e514075a7974f0e7aea87b6;m=6a0a09b2;t=6017ac0f8e15a;x=82401ca9c708c659]
    _TRANSPORT=journal
    _UID=500
    _GID=500
    _CAP_EFFECTIVE=0
    _SELINUX_CONTEXT=docker-default (enforce)
    _SYSTEMD_CGROUP=/system.slice/containerd.service
    _SYSTEMD_UNIT=containerd.service
    _SYSTEMD_SLICE=system.slice
    _SYSTEMD_INVOCATION_ID=987c8f54269e4296bf9b21f3fb6a66ab
    _BOOT_ID=2b42fb535e514075a7974f0e7aea87b6
    _MACHINE_ID=659a4c6e553b494f9510dcc33a791159
    _HOSTNAME=hsrb
    PRIORITY=3
    SYSLOG_IDENTIFIER=rosdiag
    _PID=4173
    _COMM=diagnostics
    _EXE=/opt/ros/noetic/lib/tmc_journald/diagnostics
    _CMDLINE=/opt/ros/noetic/lib/tmc_journald/diagnostics
    MESSAGE=hsrb_robot_service: present_log_power_cycle: exxx_read present_log_power_cycle failed
    ROS_NAME=hsrb_robot_service: present_log_power_cycle
    ROS_HARDWARE_ID=motor present_log_power_cycle
    ROS_TIME_STAMPS=1690476594583420753
    ROS_KEY_VALUES=11	3151
                   12	3152
                   13	3151
                   21	3149
                   22	3209
                   23	3184
                   24	3173
                   25	3179
                   31	Unknown
                   32	3180
                   41	3217
    _SOURCE_REALTIME_TIMESTAMP=1690476595437367
Fri 2023-07-28 01:50:43.713156 JST [s=834792cb7b9a47528e9f1fea693fdc95;i=133b0;b=2b42fb535e514075a7974f0e7aea87b6;m=6ceaaa8a;t=6017ac3d98232;x=77979e093fd06df2]
    _TRANSPORT=journal
    _UID=500
    _GID=500
    _CAP_EFFECTIVE=0
    _SELINUX_CONTEXT=docker-default (enforce)
    _SYSTEMD_CGROUP=/system.slice/containerd.service
    _SYSTEMD_UNIT=containerd.service
    _SYSTEMD_SLICE=system.slice
    _SYSTEMD_INVOCATION_ID=987c8f54269e4296bf9b21f3fb6a66ab
    _BOOT_ID=2b42fb535e514075a7974f0e7aea87b6
    _MACHINE_ID=659a4c6e553b494f9510dcc33a791159
    _HOSTNAME=hsrb
    PRIORITY=3
    SYSLOG_IDENTIFIER=rosdiag
    _PID=4173
    _COMM=diagnostics
    _EXE=/opt/ros/noetic/lib/tmc_journald/diagnostics
    _CMDLINE=/opt/ros/noetic/lib/tmc_journald/diagnostics
    ROS_TIME_STAMPS=1690476642863828659
    ROS_KEY_VALUES=11	3151
                   12	3152
                   13	3151
                   21	3149
                   22	3209
                   23	3184
                   24	3173
                   25	3179
                   31	Unknown
                   32	3180
                   41	3217
    MESSAGE=hsrb_robot_service: present_log_power_cycle: exxx_read present_log_power_cycle failed
    ROS_NAME=hsrb_robot_service: present_log_power_cycle
    ROS_HARDWARE_ID=motor present_log_power_cycle
    _SOURCE_REALTIME_TIMESTAMP=1690476643713156
Fri 2023-07-28 01:51:49.113994 JST [s=834792cb7b9a47528e9f1fea693fdc95;i=153ec;b=2b42fb535e514075a7974f0e7aea87b6;m=70d09a42;t=6017ac7bf71eb;x=23576924c4048751]
    _TRANSPORT=journal
    _UID=500
    _GID=500
    _CAP_EFFECTIVE=0
    _SELINUX_CONTEXT=docker-default (enforce)
    _SYSTEMD_CGROUP=/system.slice/containerd.service
    _SYSTEMD_UNIT=containerd.service
    _SYSTEMD_SLICE=system.slice
    _SYSTEMD_INVOCATION_ID=987c8f54269e4296bf9b21f3fb6a66ab
    _BOOT_ID=2b42fb535e514075a7974f0e7aea87b6
    _MACHINE_ID=659a4c6e553b494f9510dcc33a791159
    _HOSTNAME=hsrb
    PRIORITY=3
    SYSLOG_IDENTIFIER=rosdiag
    _PID=4173
    _COMM=diagnostics
    _EXE=/opt/ros/noetic/lib/tmc_journald/diagnostics
    _CMDLINE=/opt/ros/noetic/lib/tmc_journald/diagnostics
    ROS_KEY_VALUES=11	3151
                   12	3152
                   13	3151
                   21	3149
                   22	3209
                   23	3184
                   24	3173
                   25	3179
                   31	Unknown
                   32	3180
                   41	3217
    MESSAGE=hsrb_robot_service: present_log_power_cycle: exxx_read present_log_power_cycle failed
    ROS_NAME=hsrb_robot_service: present_log_power_cycle
    ROS_HARDWARE_ID=motor present_log_power_cycle
    ROS_TIME_STAMPS=1690476708266812086
    _SOURCE_REALTIME_TIMESTAMP=1690476709113994
Fri 2023-07-28 01:52:54.574472 JST [s=834792cb7b9a47528e9f1fea693fdc95;i=17435;b=2b42fb535e514075a7974f0e7aea87b6;m=74b77350;t=6017acba64af9;x=a5a22a487260c009]
    _TRANSPORT=journal
    _UID=500
    _GID=500
    _CAP_EFFECTIVE=0
    _SELINUX_CONTEXT=docker-default (enforce)
    _SYSTEMD_CGROUP=/system.slice/containerd.service
    _SYSTEMD_UNIT=containerd.service
    _SYSTEMD_SLICE=system.slice
    _SYSTEMD_INVOCATION_ID=987c8f54269e4296bf9b21f3fb6a66ab
    _BOOT_ID=2b42fb535e514075a7974f0e7aea87b6
    _MACHINE_ID=659a4c6e553b494f9510dcc33a791159
    _HOSTNAME=hsrb
    PRIORITY=3
    SYSLOG_IDENTIFIER=rosdiag
    _PID=4173
    _COMM=diagnostics
    _EXE=/opt/ros/noetic/lib/tmc_journald/diagnostics
    _CMDLINE=/opt/ros/noetic/lib/tmc_journald/diagnostics
    ROS_KEY_VALUES=11	3151
                   12	3152
                   13	3151
                   21	3149
                   22	3209
                   23	3184
                   24	3173
                   25	3179
                   31	Unknown
                   32	3180
                   41	3217
    MESSAGE=hsrb_robot_service: present_log_power_cycle: exxx_read present_log_power_cycle failed
    ROS_NAME=hsrb_robot_service: present_log_power_cycle
    ROS_HARDWARE_ID=motor present_log_power_cycle
    ROS_TIME_STAMPS=1690476773735376119
    _SOURCE_REALTIME_TIMESTAMP=1690476774574472
Fri 2023-07-28 01:54:10.229642 JST [s=834792cb7b9a47528e9f1fea693fdc95;i=194ad;b=2b42fb535e514075a7974f0e7aea87b6;m=7939dc3a;t=6017ad028b3e2;x=8addfd189293cb60]
    _TRANSPORT=journal
    _UID=500
    _GID=500
    _CAP_EFFECTIVE=0
    _SELINUX_CONTEXT=docker-default (enforce)
    _SYSTEMD_CGROUP=/system.slice/containerd.service
    _SYSTEMD_UNIT=containerd.service
    _SYSTEMD_SLICE=system.slice
    _SYSTEMD_INVOCATION_ID=987c8f54269e4296bf9b21f3fb6a66ab
    _BOOT_ID=2b42fb535e514075a7974f0e7aea87b6
    _MACHINE_ID=659a4c6e553b494f9510dcc33a791159
    _HOSTNAME=hsrb
    PRIORITY=3
    SYSLOG_IDENTIFIER=rosdiag
    _PID=4173
    _COMM=diagnostics
    _EXE=/opt/ros/noetic/lib/tmc_journald/diagnostics
    _CMDLINE=/opt/ros/noetic/lib/tmc_journald/diagnostics
    MESSAGE=hsrb_robot_service: present_log_power_cycle: exxx_read present_log_power_cycle failed
    ROS_NAME=hsrb_robot_service: present_log_power_cycle
    ROS_HARDWARE_ID=motor present_log_power_cycle
    ROS_TIME_STAMPS=1690476849373832941
    ROS_KEY_VALUES=11	3152
                   12	3153
                   13	3152
                   21	3150
                   22	3210
                   23	3185
                   24	3174
                   25	3180
                   31	Unknown
                   32	3181
                   41	3218
    _SOURCE_REALTIME_TIMESTAMP=1690476850229642
Fri 2023-07-28 01:55:15.754297 JST [s=834792cb7b9a47528e9f1fea693fdc95;i=1b4f4;b=2b42fb535e514075a7974f0e7aea87b6;m=7d21af90;t=6017ad4108738;x=68e8c94fca3abd37]
    _TRANSPORT=journal
    _UID=500
    _GID=500
    _CAP_EFFECTIVE=0
    _SELINUX_CONTEXT=docker-default (enforce)
    _SYSTEMD_CGROUP=/system.slice/containerd.service
    _SYSTEMD_UNIT=containerd.service
    _SYSTEMD_SLICE=system.slice
    _SYSTEMD_INVOCATION_ID=987c8f54269e4296bf9b21f3fb6a66ab
    _BOOT_ID=2b42fb535e514075a7974f0e7aea87b6
    _MACHINE_ID=659a4c6e553b494f9510dcc33a791159
    _HOSTNAME=hsrb
    PRIORITY=3
    SYSLOG_IDENTIFIER=rosdiag
    _PID=4173
    _COMM=diagnostics
    _EXE=/opt/ros/noetic/lib/tmc_journald/diagnostics
    _CMDLINE=/opt/ros/noetic/lib/tmc_journald/diagnostics
    MESSAGE=hsrb_robot_service: present_log_power_cycle: exxx_read present_log_power_cycle failed
    ROS_NAME=hsrb_robot_service: present_log_power_cycle
    ROS_HARDWARE_ID=motor present_log_power_cycle
    ROS_KEY_VALUES=11	3152
                   12	3153
                   13	3152
                   21	3150
                   22	3210
                   23	3185
                   24	3174
                   25	3180
                   31	Unknown
                   32	3181
                   41	3218
    ROS_TIME_STAMPS=1690476914907362222
    _SOURCE_REALTIME_TIMESTAMP=1690476915754297
Fri 2023-07-28 01:56:21.059094 JST [s=834792cb7b9a47528e9f1fea693fdc95;i=1d4d4;b=2b42fb535e514075a7974f0e7aea87b6;m=81062859;t=6017ad7f50001;x=96a18c3e859f6df8]
    _TRANSPORT=journal
    _UID=500
    _GID=500
    _CAP_EFFECTIVE=0
    _SELINUX_CONTEXT=docker-default (enforce)
    _SYSTEMD_CGROUP=/system.slice/containerd.service
    _SYSTEMD_UNIT=containerd.service
    _SYSTEMD_SLICE=system.slice
    _SYSTEMD_INVOCATION_ID=987c8f54269e4296bf9b21f3fb6a66ab
    _BOOT_ID=2b42fb535e514075a7974f0e7aea87b6
    _MACHINE_ID=659a4c6e553b494f9510dcc33a791159
    _HOSTNAME=hsrb
    PRIORITY=3
    SYSLOG_IDENTIFIER=rosdiag
    _PID=4173
    _COMM=diagnostics
    _EXE=/opt/ros/noetic/lib/tmc_journald/diagnostics
    _CMDLINE=/opt/ros/noetic/lib/tmc_journald/diagnostics
    ROS_TIME_STAMPS=1690476980196502447
    ROS_KEY_VALUES=11	3152
                   12	3153
                   13	3152
                   21	3150
                   22	3210
                   23	3185
                   24	3174
                   25	3180
                   31	Unknown
                   32	3181
                   41	3218
    MESSAGE=hsrb_robot_service: present_log_power_cycle: exxx_read present_log_power_cycle failed
    ROS_NAME=hsrb_robot_service: present_log_power_cycle
    ROS_HARDWARE_ID=motor present_log_power_cycle
    _SOURCE_REALTIME_TIMESTAMP=1690476981059094
Fri 2023-07-28 01:57:26.577603 JST [s=834792cb7b9a47528e9f1fea693fdc95;i=1f50a;b=2b42fb535e514075a7974f0e7aea87b6;m=84ede44a;t=6017adbdcbbf2;x=fbf220bea92ec1eb]
    _TRANSPORT=journal
    _UID=500
    _GID=500
    _CAP_EFFECTIVE=0
    _SELINUX_CONTEXT=docker-default (enforce)
    _SYSTEMD_CGROUP=/system.slice/containerd.service
    _SYSTEMD_UNIT=containerd.service
    _SYSTEMD_SLICE=system.slice
    _SYSTEMD_INVOCATION_ID=987c8f54269e4296bf9b21f3fb6a66ab
    _BOOT_ID=2b42fb535e514075a7974f0e7aea87b6
    _MACHINE_ID=659a4c6e553b494f9510dcc33a791159
    _HOSTNAME=hsrb
    PRIORITY=3
    SYSLOG_IDENTIFIER=rosdiag
    _PID=4173
    _COMM=diagnostics
    _EXE=/opt/ros/noetic/lib/tmc_journald/diagnostics
    _CMDLINE=/opt/ros/noetic/lib/tmc_journald/diagnostics
    ROS_KEY_VALUES=11	3152
                   12	3153
                   13	3152
                   21	3150
                   22	3210
                   23	3185
                   24	3174
                   25	3180
                   31	Unknown
                   32	3181
                   41	3218
    MESSAGE=hsrb_robot_service: present_log_power_cycle: exxx_read present_log_power_cycle failed
    ROS_NAME=hsrb_robot_service: present_log_power_cycle
    ROS_HARDWARE_ID=motor present_log_power_cycle
    ROS_TIME_STAMPS=1690477045739375352
    _SOURCE_REALTIME_TIMESTAMP=1690477046577603
Fri 2023-07-28 01:58:31.383732 JST [s=834792cb7b9a47528e9f1fea693fdc95;i=21503;b=2b42fb535e514075a7974f0e7aea87b6;m=88cac127;t=6017adfb998cf;x=f34ea338020c3ca0]
    _TRANSPORT=journal
    _UID=500
    _GID=500
    _CAP_EFFECTIVE=0
    _SELINUX_CONTEXT=docker-default (enforce)
    _SYSTEMD_CGROUP=/system.slice/containerd.service
    _SYSTEMD_UNIT=containerd.service
    _SYSTEMD_SLICE=system.slice
    _SYSTEMD_INVOCATION_ID=987c8f54269e4296bf9b21f3fb6a66ab
    _BOOT_ID=2b42fb535e514075a7974f0e7aea87b6
    _MACHINE_ID=659a4c6e553b494f9510dcc33a791159
    _HOSTNAME=hsrb
    PRIORITY=3
    SYSLOG_IDENTIFIER=rosdiag
    _PID=4173
    _COMM=diagnostics
    _EXE=/opt/ros/noetic/lib/tmc_journald/diagnostics
    _CMDLINE=/opt/ros/noetic/lib/tmc_journald/diagnostics
    ROS_KEY_VALUES=11	3152
                   12	3153
                   13	3152
                   21	3150
                   22	3210
                   23	3185
                   24	3174
                   25	3180
                   31	Unknown
                   32	3181
                   41	3218
    MESSAGE=hsrb_robot_service: present_log_power_cycle: exxx_read present_log_power_cycle failed
    ROS_NAME=hsrb_robot_service: present_log_power_cycle
    ROS_HARDWARE_ID=motor present_log_power_cycle
    ROS_TIME_STAMPS=1690477110543360710
    _SOURCE_REALTIME_TIMESTAMP=1690477111383732
Fri 2023-07-28 01:59:36.091933 JST [s=834792cb7b9a47528e9f1fea693fdc95;i=234f9;b=2b42fb535e514075a7974f0e7aea87b6;m=8ca61fc0;t=6017ae394f768;x=5782f0ad555f22ac]
    _TRANSPORT=journal
    _UID=500
    _GID=500
    _CAP_EFFECTIVE=0
    _SELINUX_CONTEXT=docker-default (enforce)
    _SYSTEMD_CGROUP=/system.slice/containerd.service
    _SYSTEMD_UNIT=containerd.service
    _SYSTEMD_SLICE=system.slice
    _SYSTEMD_INVOCATION_ID=987c8f54269e4296bf9b21f3fb6a66ab
    _BOOT_ID=2b42fb535e514075a7974f0e7aea87b6
    _MACHINE_ID=659a4c6e553b494f9510dcc33a791159
    _HOSTNAME=hsrb
    PRIORITY=3
    SYSLOG_IDENTIFIER=rosdiag
    _PID=4173
    _COMM=diagnostics
    _EXE=/opt/ros/noetic/lib/tmc_journald/diagnostics
    _CMDLINE=/opt/ros/noetic/lib/tmc_journald/diagnostics
    ROS_KEY_VALUES=11	3152
                   12	3153
                   13	3152
                   21	3150
                   22	3210
                   23	3185
                   24	3174
                   25	3180
                   31	Unknown
                   32	3181
                   41	3218
    MESSAGE=hsrb_robot_service: present_log_power_cycle: exxx_read present_log_power_cycle failed
    ROS_NAME=hsrb_robot_service: present_log_power_cycle
    ROS_HARDWARE_ID=motor present_log_power_cycle
    ROS_TIME_STAMPS=1690477175249363660
    _SOURCE_REALTIME_TIMESTAMP=1690477176091933

**Comment by yani-rl-ai** [comment]
What is the next step to fix this problem?

**Reply by hsr-admin** [reply]
Thank you for running the command.
I think there is a problem with the amplifier in head_pan_joint (jointID 31).
Please open the cover and check if the fuse is blown just to be sure.
(FYI)
https://docs.hsr.io/hsrb_maintenance_manual_en/src/check_motor_driver.html#id5

**Reply by yani-rl-ai** [reply]
thank you for reply my issue again. yes you correct. the fuse is blown. we have changed with the new fuse. but the robot still could not initialize/ wake up. we found there is a voltage in the connector and i tought by using the new fuse the board has been connected. but it still could not wake up. we check again by using the same command. 
![WhatsApp Image 2023-08-04 at 14 31 59](https://github.com/HumanSupportRobot/QandA/assets/48663658/1d1198f2-759c-4584-8ddd-0d96c0284999)

**Reply by hsr-admin** [reply]
It is possible that the amplifier is malfunctioning.
The response regarding repair may vary from location to location.
What country are you in?

**Reply by yani-rl-ai** [reply]
i am in Tokyo, Japan. Should i replace the amplifier? which procedure should i follow after this?

**Reply by hsr-admin** [reply]
There is a high possibility that the amplifier is malfunctioning.
The following is a consultation for repair in Japan.
Please refer to STEP3, although there is a charge for this service.

(Reference)
https://www.hsr.io/archives/310

**Reply by yani-rl-ai** [reply]
Thank you very much for your support. i will contact Qibitech soon
